src/Diagnostics.ModelsAndUtils/DataTableUtility.cs

```diff
@@ -0,0 +1,73 @@
+﻿using Diagnostics.ModelsAndUtils;
+using Diagnostics.ModelsAndUtils.Models;
+using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Linq;
+
+namespace Diagnostics.ModelsAndUtils
```

Comment:
If this is a utility, move this inside the utilities folder and update the namespace

Reply:
I didn't know if this was considered a "ScriptUtility" which is the only utility folder. Should I add it there or a new folder?

Reply:
Consider my other comment of creating two extension methods
response.ToDataTable()
dt.ToDataTableResponse()

Perhaps then you dont need a separate file. Just move this class into DataTableResponseObject class file

Reply:
The response.ToDataTable() I am not clear as to what that would be for. I think we need that for changing something like a list of objects into a datatable but I don't believe it is needed for converting between DataTable and DataTableResponseObject, since as I mentioned in the other comment, that does not happen inside the script